chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -111,9 +111,10 @@ class CurrentUser(
             syncCurrentUser.getReadCursor(room.id, user.id)
 
     /**
-     * Make the users listed in [userIds] members of the room identified by [roomId]. This does not mean
-     * the user(s) will see new messages immediately, it means the room will be in their list of rooms
-     * and they will need to subscribe themselves to see any updates.
+     * Make the users listed in [userIds] members of the room identified by [roomId].
+     * For any users currently only they will be notified with a [AddedToRoom] event and you will need to handle
+     * subscribing to the room to see any new messages sent to the room. The current users list of rooms
```

Comment:
It's not clear who (*they*, not you, writing in this context) would do the subscribing